wiki/content/enterprise-features/index.md

```diff
@@ -703,3 +702,47 @@ Here's an example to run bulk loader with a key used to write encrypted data:
 ```bash
 dgraph bulk --encryption_key_file "./enc_key_file" -f data.json.gz -s data.schema --map_shards=1 --reduce_shards=1 --http localhost:8000 --zero=localhost:5080
 ```
+
+## Encrypted Backups
```

Comment:
I think we dont need a separate section for Encrypted Backups. The documentation for this feature can be inserted into the binary backups topic elsewhere in this doc.